### DISCUSSION TITLE
Unable to close lightbox in mobile devices

### DISCUSSION THREAD

**Discussion by haendzel** [biati-digital/glightbox#541] [Q&A]
Hey everyone,

I'm having a problem where even though I've enabled the option `closeOnOutsideClick: true` in glightbox, it's still not working on mobile. The only way to close it is by clicking the close button.

I've been messing around with this for ages, trying all sorts of different things, but no luck. Has anyone else managed to get this lightbox to close properly?

```
    const lightbox = GLightbox({
      touchNavigation: true,
      closeOnOutsideClick: true,
    });
```

**Comment by graphixela** [comment]
Same problem
